rllib/core/rl_module/torch/torch_rl_module.py

```diff
@@ -39,3 +45,37 @@ def is_distributed(self) -> bool:
         """Returns True if the module is distributed."""
         # TODO (Avnish): Implement this.
         return False
+
+
+class DDPRLModuleWrapper(DDP, RLModule):
```

Comment:
why not simply TorchDDPRLModule?
RLModule by-design can hold other RLModules right? So Wrapper may not be necessary.

Reply:
+1

Reply:
This is just a naming suggestion right? I am changing the name to TorchDDPRLModule, but if I miss-understood your comment, let me know.

Reply:
Yep. Thanks.